### PR TITLE
Tagging some Leo tests as SmokeTest

### PR DESCRIPTION
**Pull request by aweng98**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by aweng98** [comment]
leo-build failed.

jenkins retest

**Comment by aweng98** [comment]
jenkins retest

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterMonitoringSpec.scala [hunk 1]
```diff
@@ -196,7 +197,7 @@ class ClusterMonitoringSpec extends FreeSpec with LeonardoTestUtils with Paralle
     //Test to check if extensions are installed correctly
     //Using nbtranslate extension from here:
     //https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/nbTranslate
-    "should install a single user specified notebook extension" in {
+    "should install a single user specified notebook extension" taggedAs Tags.SmokeTest in {
```
Comment (rtitle) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterMonitoringSpec.scala]: I might remove this as a smoke test -- the extension functionality is not widely used, and should be exercised anyway with the JupyterLab test.

**Comment by rtitle** [comment]
I'd probably recommend adding "should pause and resume a cluster" as a smoke test and removing "should install a single user specified notebook extension". Otherwise, this looks good.

**Comment by aweng98** [comment]
jenkins retest

**Comment by aweng98** [comment]
leo-build failed

jenkins retest

**Comment by aweng98** [comment]
jenkins retest

**Pull request merged**